### PR TITLE
Added login, signup, logout and quiz page. Please review!

### PR DESCRIPTION
**Pull request by muskanbagrecha**

### CODE REVIEW

File: css/quiz.css [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    align-items: center;
+    padding: 2rem 0 0 0;
+    width: 100%;
+    /* background-color: red; */
+}
+
+.quiz__options{
+    width: 100%;
+    padding: 0;
+}
```
Comment (ShrutiLakhani) [inline css/quiz.css:41]: Your code is clean and crisp. Good use of Semantic HTML and utility classes.

**Review by ShrutiLakhani — APPROVED** [review]
Good job. UI is smooth. Everything looks good to me

**Review by kushalsourav — APPROVED** [review]
Everything looks clean, good to merge

**Comment by shruthipatil11** [comment]
Semantics, followed, well structured .Code is good to merge

File: css/quiz.css [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+.quiz__container{
+    width: 80%;
+    max-width: 40rem;
+}
+
+.quiz__content{
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
```
Comment (kushalsourav) [inline css/quiz.css:23]: clean code

**Review by m26unkwn — APPROVED** [review]

**Review by kushalsourav — APPROVED** [review]

**Review by shruthipatil11 — APPROVED** [review]

**Review by m26unkwn — COMMENTED** [review]
1. Naming Convention is good. 
2. Easy to read
3. Clean UI is superb
4. Images are beautiful.

**Pull request merged**